packages/lib/src/FormControllerClass/FormControllerClass.ts

```diff
@@ -44,28 +43,39 @@ export class FormControllerClass {
     this.fieldLevelValidations[fieldName] = onValidateFunction;
   };
 
-  @action removeFieldLevelValidation = (fieldName: string) => {
-    delete this.fieldLevelValidations[fieldName];
-  };
-
   protected safeApiValuesCopy: Record<string, unknown> = {};
 
-  @action protected updateAPIValues = (fieldName: string, value: unknown) => {
-    const safeValue = toJSCompat(value, false);
-    const {onFormat} = this.options;
-    const field = this.fields.get(fieldName);
+  @action protected updateAPIValues = (fieldName?: string, value?: unknown) => {
+    if (fieldName) {
+      const safeValue = toJSCompat(value, false);
+      const field = this.fields.get(fieldName);
 
-    this.options.fieldValueToFormValuesConverter.set(
-      this.safeApiValuesCopy,
-      fieldName,
-      field && field.fieldProps?.onFormat && field.meta.isMounted ? field.fieldProps.onFormat(safeValue) : safeValue,
-    );
+      this.options.fieldValueToFormValuesConverter.set(
+        this.safeApiValuesCopy,
+        fieldName,
+        field && field.fieldProps?.onFormat && field.meta.isMounted ? field.fieldProps.onFormat(safeValue) : safeValue,
+      );
+    } else {
+      this.safeApiValuesCopy = {};
+
+      this.fields.forEach((field, name) => {
+        if (field.meta.isMounted) {
```

Comment:
you check `field.meta.isMounted` for `fields` scenario, should you check the same for single field update scenario?